src/registrar/models/domain.py

```diff
@@ -1651,7 +1655,7 @@ def _get_dnssec_data(self, response_extensions):
         return dnssec_data
 
     def _update_hosts_and_contacts(self, cleaned, fetch_hosts, fetch_contacts):
-        """Capture and store old hosts and contacts from cache if they don't exist"""
+        """Capture and cache old hosts and contacts from cache if they don't exist in cleaned"""
```

Comment:
"""Update cache from old cache"""

Is this what the method's doing?

Reply:
You're right, I am updating the comment before merging